### ISSUE TITLE
BLT doesn't show up, even when installed correctly

### ISSUE THREAD

**Issue by TheMTNDewd** [JamesWilko/Payday-2-BLT#169]
No matter what I do, BLT doesn't seem to install correctly. When I go to the main menu, theres no mod options, or anything.

Yes I installed it correctly. Yes I downloaded it from GitHub. Yes I installed the latest version. Yes I tried uninstalling and reinstalling. Yes I tried 2.2 instead of 2.3. Yes I did a fresh install of payday 2, and then installed it. Yes I've verified my game files on steam. Yes I launched the game with the exe. Yes I tried running it as administrator. Yes I have C++ 2013 x86 runtime installed. Yes I repaired it. Yes I uninstalled, and reinstalled that. Yes I've tried to use the mods folder and .dll from someone with a working copy

I've been dealing with this issue for about a week, and I'm at wits end. I've read every forum post, reddit thread, youtube video, and nothing seems to work. I've found a few others in my situation. So what is the issue? What information can I provide (the blt logs are always empty, it seems like it just doesn't load up) to help you find the issue, or what else can I do/try to fix this?

**Comment by clydejallorina** [comment]
Are you sure that the BLT's DLL file is also installed into the Payday 2 folder?

**Comment by TheMTNDewd** [comment]
Yes. I've even used other people's dlls since they claimed it would fix the issue

**Comment by TryxxStyxx6** [comment]
Random idea: try installing 2.0 then just updating through in game if it works.

**Comment by TheMTNDewd** [comment]
Whenever I install it, it doesn't show up in game. There's no BLT options or anything. So there would be no way to try that

**Comment by Vlad-00003** [comment]
@TheMTNDewd, show the screenshot of your game directory so we can be sure that this is not a case of a wrong installation.

**Comment by Sickerinopack** [comment]
Go to your PAYDAY 2 folder, right-click on the .exe, select 'Properties', go to 'Compatibility' and make sure that 'Disable fullscreen optimization' IS unchecked. That fixed the problem for me.

**Comment by TheMTNDewd** [comment]
@Vlad-00003 
![untitled](https://user-images.githubusercontent.com/33240538/32417983-4dd266ec-c227-11e7-8463-1fc7dcbbbc8f.png)

@Sickerinopack Thank you, this is one of the first suggestions I haven't seen before, but unfortunately 'Disable fullscreen optimization' was already unchecked when I went to look.

**Comment by BaLbIHa** [comment]
@Sickerinopack Thanks it's work to me too, but if i disable fullscreen optimization. i have tearing effect on my screen.

**Comment by Mistercrit** [comment]
I am also having the same exact issue you are having regarding BLT not working even though it is properly installed. I'm close to giving up on playing Payday at this point if I can't get it fixed soon.
![b2c16fd1c8070ab0f97d204f49a839cb](https://user-images.githubusercontent.com/29848604/32590347-8e8a54b4-c4df-11e7-9fa5-9f80f8503db9.png)

**Comment by TheTakkle** [comment]
Same for me. BLT didn't work.

**Comment by mkSolas** [comment]
Are you running the latest version of Windows 10? I feel like that's the problem since I just upgrade from Windows 7 to Windows 10 and BLT all of a sudden stopped working.
I have Windows 10 Pro Version 1709 OS 16299.19.

**Comment by BaLbIHa** [comment]
@mkSolas  I hate so much 1709 update, i'm downgrade to 1607, because in 1709 version, game crashing and screen have tearing effect.

**Comment by mkSolas** [comment]
Yup, I'd figure it was the (could be also the creator's update) Fall Creator's update for Windows 10, but oddly enough it works fine for my friend who also has the same update. I also tried using his files, but it didn't work. Sadly enough I can't rollback anymore it seems :(

**Comment by DaKashi-bmp** [comment]
Happens the same to me, with the latest Windows update, BLT doesn't even work.

EDIT: Unchecking Compatibility mode on options fixed it for me.

**Comment by dmr8888** [comment]
I fixed the problem by running the game's .exe directly instead of running from the Steam client.

**Comment by Mistercrit** [comment]
I've tried unchecking compatibility mode on the .exe, but it turned out I didn't even have it on in the first place.

I've also tried running the game .exe directly from the installation folder instead of running it through Steam and I get the same results, the mod options doesn't show up.
